tokio/src/process/unix/mod.rs

```diff
@@ -62,9 +63,7 @@ impl Kill for StdChild {
     }
 }
 
-lazy_static::lazy_static! {
-    static ref ORPHAN_QUEUE: OrphanQueueImpl<StdChild> = OrphanQueueImpl::new();
-}
+static ORPHAN_QUEUE: Lazy<OrphanQueueImpl<StdChild>> = Lazy::new(|| OrphanQueueImpl::new());
```

Comment:
```suggestion
static ORPHAN_QUEUE: Lazy<OrphanQueueImpl<StdChild>> = Lazy::new(OrphanQueueImpl::new);
```

Is a lambda necessary here?

Reply:
In fact, this is necessary to pass clippy.

Reply:
Done :)